util/log/log.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: Tamir Duberstein (tamird@gmail.com)
+
+syntax = "proto3";
+package cockroach.util.log;
+option go_package = "log";
+
+import weak "gogoproto/gogo.proto";
+
+// Entry represents a cockroach structured log entry.
+message Entry {
+  int64 severity = 1 [(gogoproto.casttype) = "Severity"];
+  // Nanoseconds since the epoch.
+  int64 time = 2;
```

Comment:
I just ported the existing structure from the log package. Feel free to change this, it should be pretty easy.